src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { Profile } from "./Profile";
+import { useState } from "react";
```

Comment:
You already have an import from `React` in line 1. If you need to do both a named and a default import from the same path, you can do it like this:

```js
import React, { useState } from 'react';
```